### PR TITLE
New Medieval 2 Total War Cheat Sheet

### PR DESCRIPTION
**Pull request by yasharmaster**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Medieval 2 Total War Cheat Sheet**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

What changes does this PR introduce? 

###### Related Issues or Discussions 

What issues or discussions (if any) does this PR address? Please provide the issue # or link.

###### People to notify

@mention any relevant people/organizations
@tagawa 

---

Instant Answer Page: https://duck.co/ia/view/medieval_2_total_war_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html):

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/medieval-2-total-war.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    "section_order": [
+        "Console codes",
+        "Adding Units to Stacks"
+    ],
+
+    "sections": {
+        "Console codes": [
+            {
+                "key": "character_reset",
+                "val": "Allows a character to move again."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/medieval-2-total-war.json]: Lets remove periods from end of val fields

**Comment by yasharmaster** [comment]
@jophab Removed periods from val fields.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=medieval 2 total war cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/medieval_2_total_war_cheat_sheet_mobile.png?SFYBfKhpcX)

**Comment by gautamkrishnar** [comment]
LGTM :+1:

**Comment by daxtheduck** [comment]
@yasharmaster this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**